docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml

```
@@ -30,8 +30,10 @@ content: |
     
   % endfor
 ---
-# The closing '}}' doesn't really need to be contained.
-# It just seemed more consistent
+comment: |
+  The setup interview adds this file to the repo during GitHub test setup.
```

```suggestion
  The setup interview adds this file (run through mako) to the repo during GitHub test setup.
```

Developers can't quite copy-paste this file, due to the large number of `${ '${{' }` in it.

---

```suggestion
  The setup interview adds this file (run through mako) to the repo during GitHub test setup. If you're creating this file manually, make sure to remove the Mako.
```